config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -9,50 +9,113 @@ spec:
   nodeselector: {}
   bytecode:
     image:
-      url: quay.io/bpfman-bytecode/go-app-counter:latest
+      url: quay.io/bpfman-bytecode/app-test:latest
+      # imagepullpolicy: Always
   programs:
-    - type: Kprobe
+    - bpffunctionname: kprobe_test
+      type: Kprobe
```

Comment:
Just wondering, is case significant here? It seems to me that `kprobe` or `KPROBE` or `Kprobe` should probably work. I'd prefer to use the lowercase version.

Reply:
Yes and no.  Left unchecked, I don't think Kubernetes would care, but it is case sensitive in that kubernetes passes these values through to the controller code in the case that the user enters them.  However, we also have validation logic that only allows one case, so in our case (no pun intended) it is case sensitive.  If we wanted to support case insensitivity, we'd have to handle it in our code.

I made everything lower case.  We can disuss whether to use camelCase instead.

Reply:
I just went back and changed all of the json tags to camelCase